test/parallel/test-async-wrap-check-providers.js

```diff
@@ -1,6 +1,11 @@
 'use strict';
 
 const common = require('../common');
+if (!common.hasCrypto) {
+  common.skip('missing crypto');
+  return;
+}
+
```

Comment:
Perhaps the `crypto` AsyncWrap test can merely be isolated instead of aborting the entire test?